### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by BelaM80**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,12 +22,15 @@
   "homepage": "https://github.com/BelaM80/webpack-demo#readme",
   "devDependencies": {
     "clean-webpack-plugin": "^3.0.0",
+    "concurrently": "^5.3.0",
```
Comment (lmachens) [inline package.json:25]: `concurrently` is a nice module, but you don't use it here.
Same with `live-server`. I would uninstall bot modules.

File: package.json [hunk 2]
```diff
@@ -20,12 +22,15 @@
...
     "html-webpack-plugin": "^4.5.0",
+    "live-server": "^1.2.1",
     "style-loader": "^2.0.0",
     "webpack": "^5.1.3",
-    "webpack-cli": "^4.0.0"
+    "webpack-cli": "^4.0.0",
+    "webpack-dev-server": "^3.11.0"
   },
   "dependencies": {
     "lodash": "^4.17.20"
```
Comment (lmachens) [inline package.json:36]: You already removed `lodash` import, but the module is still installed.

**Review by lmachens — COMMENTED** [review]
Good job!

File: package.json (hunk 1 above)
Comment (BelaM80) [reply package.json:25]: thanks, I will do that

**Pull request merged**